### PR TITLE
Add skip existing reinforcements flag to ctar

### PR DESCRIPTION
**Pull request by Diet-Cola**
Admin love

### CODE REVIEW

**Comment by Diet-Cola** [comment]
Implements #320

**Comment by RedDevel2** [comment]
Omg bless you

File: plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/command/AreaReinforce.java [hunk 1]
```diff
@@ -65,6 +65,10 @@ public void execute(Player p, @Optional String targetGroup, String minX, String
 			CitadelUtility.sendAndLog(p, ChatColor.RED, "One of the arguments you provided was not a number");
 			return;
 		}
+		boolean skipReins = false;
+		if (skipReinforcements.equalsIgnoreCase("true")) {
+			skipReins = true;
+		}
```
Comment (okx-code) [inline plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/command/AreaReinforce.java:71]:
```suggestion
		boolean skipReins = skipReinforcements.equalsIgnoreCase("true");
```

File: plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/command/AreaReinforce.java [hunk 2]
```diff
@@ -74,6 +78,9 @@ public void execute(Player p, @Optional String targetGroup, String minX, String
 						continue;
 					}
 					Block protecting = ReinforcementLogic.getResponsibleBlock(current);
+					if (skipReins) {
+						continue;
+					}
```
Comment (okx-code) [inline plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/command/AreaReinforce.java:83]: Doesn't this just skip all reinforcements?

Comment (Diet-Cola) [reply plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/command/AreaReinforce.java:80]: No? We do multiple checks on the block first i.e if its air etc before creating the reinforcement

Comment (Diet-Cola) [reply plugins/citadel-paper/src/main/java/vg/civcraft/mc/citadel/command/AreaReinforce.java]: my bad, good catch

**Pull request merged**